wsimod/nodes/wtw.py

```diff
@@ -6,7 +6,7 @@
 """
 from wsimod.core import constants
 from wsimod.nodes.nodes import Node, Tank
-
+from typing import Any, Dict
 
 class WTW(Node):
     """"""
```

Comment:
Take this opportunity to explain what `WTW` is - and even better if you provide a readable name and not just an acronym. End users might know what it is but developers like myself and potential future contributors have no clue what you are talking about. 

And the same about WWTW!